backend/src/Service/CaptainFinancialSystem/CaptainFinancialSystemAccordingToCountOfHoursService.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Service\CaptainFinancialSystem;
+
+use App\AutoMapping;
+use App\Entity\CaptainFinancialSystemAccordingToCountOfHoursEntity;
+use App\Manager\CaptainFinancialSystem\CaptainFinancialSystemAccordingToCountOfHoursManager;
+use App\Response\CaptainFinancialSystem\CaptainFinancialSystemAccordingToCountOfHoursResponse;
+
+class CaptainFinancialSystemAccordingToCountOfHoursService
+{
+    private CaptainFinancialSystemAccordingToCountOfHoursManager $CaptainFinancialSystemAccordingToCountOfHoursManager;
```

Comment:
Please update the variable name, and check all places in which it is being used.